akka-actor-tests/src/test/scala/akka/actor/TypedActorSpec.scala

```diff
@@ -72,7 +72,7 @@ object TypedActorSpec {
 
     @silent
     @throws(classOf[TimeoutException])
-    def self = TypedActor.self[Foo]
+    def self = akka.actor.TypedActor.self[Foo]
```

Comment:
not that I care about this test, but why is this needed? isn't it in the same package?

Reply:
In this file, Scala 3 is complaining about the ambiguous reference to `TypedActor`.
In between this `object` and a local `class` that comes from somewhere.